### PR TITLE
UV mapping

### PR DESCRIPTION
**Pull request by ebspark**
Skill issue, get better +L+bozo+youre+ unoriginal +a human did it first

### CODE REVIEW

File: sgm2obj.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import struct, os, argparse
 
 def main():
+
```
Comment (twhlynch) [inline sgm2obj.py:4]: epic important changes

File: sgm2obj.py [hunk 2]
```diff
@@ -121,8 +119,12 @@ def write_obj(meshes, materials, filename):
...
+                else:
+                  f.write(f'vt {v[2][0][0]} {1-v[2][0][1]}\n')
             for i in range(0, len(indices), 3):
-                f.write(f'f {indices[i] + 1}//{indices[i] + 1} {indices[i + 1] + 1}//{indices[i + 1] + 1} {indices[i + 2] + 1}//{indices[i + 2] + 1}\n')
+                f.write(f'f {indices[i] + 1}/{indices[i] + 1}/{indices[i] + 1} {indices[i + 1] + 1}/{indices[i + 1] + 1}/{indices[i + 1] + 1} {indices[i + 2] + 1}/{indices[i + 2] + 1}/{indices[i + 2] + 1}\n')
 
 if __name__ == '__main__':
-    main()
\ No newline at end of file
+    main()
```
Comment (twhlynch) [inline sgm2obj.py:130]: :bigbrain:

**Review by twhlynch — APPROVED** [review]
garbage

File: sgm2obj.py (hunk 1 above)
Comment (ebspark) [reply sgm2obj.py:4]: I agree

**Pull request merged**